shin/src/layer/message_layer/messagebox.rs

```diff
@@ -159,16 +185,14 @@ impl Renderable for Messagebox {
             return;
         }
 
-        let height = 360.0;
-
         let transform = transform
             * Matrix4::from_translation(Vector3::new(
                 -960.0,
-                -540.0 + (1080.0 - height) - 32.0,
+                -540.0 + (1080.0 - self.metrics.height) - 32.0,
                 0.0,
             ));
         // TODO: do not upload the vertices if they haven't changed
-        let vertices = build_vertex_buffer(0.0, height);
+        let vertices = build_vertex_buffer(self.metrics);
```

Comment:
I expect the height to be dynamic (as it is in the game), so I don't think it makes sense to take it from `MessageMetrics`

Reply:
alright, makes sense, I made a separate struct field `dynamic_height` that can be changed on its own. Both this and especially the `metrics.height` are pretty meaningless at this time, but I think it would make sense to keep them for the future